package io.github.vinifillos.msavaliadorcredito.application.exception;

public class DadosClienteNotFoundException extends Exception{
    public DadosClienteNotFoundException() {
        super("Dados do cliente n√£o encontrado para o cpf informado");
    }
}
